INSTALL.md

```diff
@@ -8,6 +8,7 @@ enclave creation to work.
 # apt install docker.io
 # apt install curl
 # apt install protobuf-compiler
+# apt install libprotobuf-dev
```

Comment:
I am generally not a big fan of this apt-get way of installing deps, since you never know which version you are going to actually get, and different people on different machines, or even on the same machine at different time, may get incompatible versions and we will get cryptic error messages due to subtle parsing differences etc. Perhaps for the time being specify somewhere which versions you installed so that if someone has issues they could at least compare their version against yours? Note that Docker (if used well) or Nix solve exactly this problem, so I think we should start relying on either of them . Also FYI this prevents any hope of actually achieving reproducible builds.